EDAM_dev.owl

```diff
@@ -14518,11 +14518,11 @@
     <owl:Class rdf:about="http://edamontology.org/data_2101">
         <rdfs:subClassOf rdf:resource="http://edamontology.org/data_2118"/>
         <created_in>beta12orEarlier</created_in>
-        <oboInOwl:hasDefinition>An identifier of a software end-user (typically a person).</oboInOwl:hasDefinition>
+        <oboInOwl:hasDefinition>Authentication data usually used to logg into an account in order to use a software or consult a website/data base.</oboInOwl:hasDefinition>
```

Comment:
Nice! Much improved 👍🏽

A separate issue for fixing the spelling and checking all the authentication data concepts: #832.